src/types/Producer.ts

```diff
@@ -0,0 +1,20 @@
+export type Producer = {
```

Comment:
@kevwhuang I want to populate a test DB with some data for the next step. Does this look good for a starting datatype? We can add and remove fields easily even as the DB grows so it doesn't have to be perfect.